server/lib/genome/importers/file_importers/nci.rb

```diff
@@ -1,125 +1,55 @@
 module Genome; module Importers; module FileImporters; module Nci;
   class Importer < Genome::Importers::Base
     attr_reader :file_path
-    attr_reader :source_db_name
-    attr_reader :source
-    attr_reader :drug_filter
+    attr_reader :worksheet
 
     def initialize(file_path)
-      @file_path = handle_file_location file_path
-      @source_db_name = 'NCI'
+      @file_path = file_path
     end
 
-    def default_filetype
-      'xml'
+    def get_version
+      source_db_version = Date.today.strftime('%d-%B-%Y')
+      @new_version = source_db_version
     end
 
-    def run_parser
-      @drug_filter = DrugFilter.new
-      parser = Nokogiri::XML::SAX::Parser.new(drug_filter)
-      parser.parse(File.open(@file_path))
+    def import
+      remove_existing_source
+      create_new_source
+      create_interaction_claims
```

Comment:
this method name should be changed to `create_claims` and the `remove_existing_source` and `create_new_source` lines should be deleted